### DISCUSSION TITLE
POI with stops

### DISCUSSION THREAD

**Discussion by dicarsio** [bimmerconnected/bimmer_connected#288] [General] [answered]
Hi,

Do you guys know how to send multiple POIs (with stops) like we could do on the old app? Forgot to sniff out the app before the 30th June and they disabled it.

Thanks

![8EF96D39-4E04-48AF-93DF-DBFBCCEE90F1](https://user-images.githubusercontent.com/260523/124061348-9fd3c380-da26-11eb-9fd4-033ce1ac45e4.jpeg)

**Comment by rikroe** [comment, marked as answer]
Haven't really tried it. But you could try a couple of things out:
- Get a token using `bimmer_connected cli`
- Use this token e.g. in Postman (get the required URL and headers from `account.py`).

The data structure is currently sent like this: https://github.com/bimmerconnected/bimmer_connected/blob/7f89bfaab1dcc5696e7cb2c99284d36dd203ea22/bimmer_connected/remote_services.py#L106-L111
Maybe try multiple POIs with `poi1`, `poi2` or sending a list of POIs.

Haven't tried it, but can you maybe just change the time of your phone and try it out? The APIs are still running (otherwise this whole library would have a massive problem)....

**Reply by dicarsio** [reply]
Tried to change the date, but clicking the map function crashes the app (iOS). Does it work for you?

Do you have the chance to sniff out the request?

Thanks

**Reply by rikroe** [reply]
On Android, changing the date works. But no idea how I am supposed to add intermediate stops - I can just send one destination at a time...

**Reply by dicarsio** [reply]
If memory doesn’t fail me, you need to input one destination (without sending) then search for another and click on “add as intermediary”.

**Reply by rikroe** [reply]
Tried that, but wasn't able to do as described. Maybe that function never was available on Android?
Wouldn't surprise me given the love BMW gave the Android app from the start.

Sorry for not being able to help you here.

**Reply by dicarsio** [reply]
@rikroe no worries at all! thanks for all the help though. I'm sad I didn't have time to mitm the requests and see how it was being done :(. anyway, how do you bypass the "time to change" splash screen? it appears even if I select a date mid-june. Thanks!

**Reply by rikroe** [reply]
Just changed the time in Android system settings to June 29th (and of course disabled automatic time).
Had to reset my phone though so don't have app installed anymore - could try to get it from APKpure or so to test if it still works though.
